subsys/bluetooth/host/audio/media_proxy.h

```diff
@@ -358,10 +358,23 @@ struct media_proxy_ctrl_cbs {
 	 *                   or errno on negative value.
 	 * @param position   The player's position in the track
 	 */
-	void (*track_position)(struct media_player *player, int err, int32_t position);
+	void (*track_position_recv)(struct media_player *player, int err, int32_t position);
 
 	/**
-	 * @brief Playback Speed callback
+	 * @brief Track Position write callback
+	 *
+	 * Called when the Track Position is written
```

Comment:
The "read" callbacks have this text:

```
	 * @brief Track Position receive callback
	 *
	 * Called when the Track Position is read or changed
```

Which is common for both the client and server, right? 

Unless we are expecting to use this callback for when the server updates a value itself (like setting the track position by turning a knob), I think this should be made clear that is a client operation on a remote device.

Reply:
The write callbacks (as the receive callbacks) will be called for both local and remote players.
This is on purpose, to unify the behaviour.

Would it be clearer if the wording was changed to "when the Track Position is set"?
("set" is used for the commands to set values, locally and remote, while "write" may be interpreted as being a GATT write, i.e. a remote write.)

Reply:
You can't use `set` for control point operations though, so I think it's better to just keep it as "write". 

But if this callback is for both local and remote players, then we can keep it as is.